primitives/chain-westend/src/lib.rs

```diff
@@ -62,57 +62,24 @@ impl ChainWithGrandpa for Westend {
 	const AVERAGE_HEADER_SIZE_IN_JUSTIFICATION: u32 = AVERAGE_HEADER_SIZE_IN_JUSTIFICATION;
 }
 
-/// `AssetHubWestend` parachain definition
-#[derive(Debug, Clone, Copy)]
-pub struct AssetHubWestend;
-
-// AssetHubWestend seems to use the same configuration as all Polkadot-like chains, so we'll use
-// Westend primitives here.
-impl Chain for AssetHubWestend {
-	const ID: ChainId = *b"ahwe";
-
-	type BlockNumber = BlockNumber;
-	type Hash = Hash;
-	type Hasher = Hasher;
-	type Header = Header;
-
-	type AccountId = AccountId;
-	type Balance = Balance;
-	type Nonce = Nonce;
-	type Signature = Signature;
-
-	const STATE_VERSION: StateVersion = StateVersion::V0;
-
-	fn max_extrinsic_size() -> u32 {
-		Westend::max_extrinsic_size()
-	}
-
-	fn max_extrinsic_weight() -> Weight {
-		Westend::max_extrinsic_weight()
-	}
+parameter_types! {
+	pub const SS58Prefix: u8 = 42;
```

Comment:
@bkontur I haven't found any references to that (in this repo and in `polkadot-sdk` - maybe you know where it is used? Otherwise - let's remove it?